documentation/modules/deploying/proc-deploy-kafka-cluster.adoc

```diff
@@ -68,20 +67,21 @@ spec:
 
 .Procedure
 
-. Create and deploy an _ephemeral_ or _persistent_ cluster.
+. Create and deploy an ephemeral or persistent cluster.
 +
-For development or testing, you might prefer to use an ephemeral cluster.
-You can use a persistent cluster in any situation.
-
-** To create and deploy an _ephemeral_ cluster:
+For development or testing, you might prefer to use an ephemeral cluster. You can use a persistent cluster in any situation.
```

Comment:
I would not say `For development or testing, you might prefer to use an ephemeral cluster`. If we want to touch the epheral storage, we should say it is only good for a short runnign clusters such as in CIs etc.

Reply:
I've removed all of this. We talk about the concepts of ephemeral and persistent in the intro anyway.